src/providers/MeetingProvider/docs/MeetingProvider.stories.mdx

```diff
@@ -91,11 +93,52 @@ interface PostLogConfig = {
 ```
 
 #### simulcastEnabled
+
 To enable simulcast for the meeting pass the `simulcastEnabled` prop with value set to `true`. By default, it is `false`.
 For more information on Simulcast, check Amazon Chime JS SDK [simulcast guide](https://github.com/aws/amazon-chime-sdk-js/blob/master/guides/05_Simulcast.md).
 
+#### logger
+
+The `Logger` object that you want to be used in the meeting session.
+If you pass in a `Logger` object using this parameter, the `MeetingManager` will use this object instead of creating a logger based on `logLevel` and `postLogConfig` to initialize the meeting session.
+
+#### videoDownlinkBandwidthPolicy
+
+The `VideoDownlinkBandwidthPolicy` object that you want to be used in the meeting session.
+For more information on `VideoDownlinkBandwidthPolicy`, check Amazon Chime JS SDK [priority downlink policy guide](https://aws.github.io/amazon-chime-sdk-js/modules/prioritybased_downlink_policy.html).
+
+For example, to use the `VideoPriorityBasedPolicy` in the meeting:
+
+```jsx
+import {
+  LogLevel,
+  ConsoleLogger,
+  VideoPriorityBasedPolicy,
+} from 'amazon-chime-sdk-js';
+import { MeetingProvider } from 'amazon-chime-sdk-component-library-react';
+
+const Root = () => {
+  const logLevel = LogLevel.info;
+  const logger = new ConsoleLogger('SDK', logLevel);
+  const videoDownlinkBandwidthPolicy = new VideoPriorityBasedPolicy(logger);
+
+  const meetingConfig = {
+    logLevel,
```

Comment:
We should not pass logLevel in this example since it is not used.

Reply:
Actually the problem is logLevel is not optional 😅

Reply:
Yes, @devalevenkatesh is right

Reply:
If i read the code right it is auto default to LogLevel.WARN in meeting provider anyway right? The required parameter is with the MeetingManager constructor.

Reply:
Oh yes, you are right. Thanks for pointing out, I will update